help/tips/2.txt

```diff
@@ -0,0 +1,2 @@
+<<You can buy or sell items from players shops.
```

Comment:
Can these be named `02.txt` for proper sorting?

Also splitting these into a separate commit in the same PR would be nice.